src/purging_rmw.c

```diff
@@ -246,16 +246,30 @@ get_then_time(struct dirent *entry, const char *entry_path)
  */
 int
 purge (const st_waste * waste_curr)
+{
+  return empty_option_purge(waste_curr, 0);
+}
+
+/*!
+ * Purges files older than x number of days, unless purge_after is set to
+ * 0 in the config file.
+ * @param[in] waste_curr the linked list of waste folders
+ * @param[in] empty_trash forces the trash to be completely emptied
+ * @return error number
+ * @see is_time_to_purge
+ * @see get_then_time
+ */
+int
+empty_option_purge (const st_waste * waste_curr, bool cmd_empty)
```

Comment:
This wrapper function is not good for code clarity. You see how the function is now named "empty_option_purge" but it contains the actual code that does the purging, which means the function name doesn't actually match what the function does.